com.unity.render-pipelines.universal/Editor/ShaderGraph/Includes/PBRForwardPass.hlsl

```diff
@@ -114,6 +114,14 @@ half4 frag(PackedVaryings packedInput) : SV_TARGET
         surface.clearCoatSmoothness = saturate(surfaceDescription.CoatSmoothness);
     #endif
 
+    #if defined(_ALPHAMODULATE_ON)
+    // Fake alpha for multiply blend by lerping albedo to 1.
+    // Manual adjustment for less multiply effect (sort of "premultiplied alpha" version)
+    // would be painting whiter pixels in the texture.
+    // This emulates that procedure by whitening the "base color" automatically using the alpha value.
+    surface.albedo = lerp(1, surface.albedo, surface.alpha);
+    #endif
+
```

Comment:
This code seems to be duplicated in multiple places. 
Could we extract this in to a function (ShaderVariablesFunctions.hlsl) and call that instead?

Reply:
Yes, it's a one _code_ line though. I could cut the comments from all except one.
I think there's AlphaModulate() function for particles, but that does bake other things in it.
I'll look into it if I could modify that to be used here.